Makefile

```diff
@@ -15,7 +15,7 @@ ifndef PULL_SECRET
 endif
 
 LIBVIRT_IMAGE_PATH := $(or ${LIBVIRT_IMAGE_PATH},/var/lib/libvirt/images)
-BASE_IMAGE_PATH_SNO = $(LIBVIRT_IMAGE_PATH)/sno-test.qcow2
+BASE_IMAGE_PATH_SNO = $(LIBVIRT_IMAGE_PATH)/sno1.qcow2
```

Comment:
better use a variable for the name so if we want to change again, we just change in one spot ...

Reply:
Separate PR, as it takes too much effort for me to test any changes